adds supports for invoking a Lua script on the end of incoming AXFR zone transfer (#10052)
PowerDNS/pdns: docs/settings.rst

Script to be used to edit incoming AXFRs, see :ref:`modes-of-operation-axfrfilter`

.. _setting-lua-axfr-end-script:

``lua-axfr-end-script``
-------------------

-  String
-  Default: empty

> Script to be used on AXFR transfer end, see :ref:`modes-of-operation-axfrend`

Reviewer: ```suggestion
Script to be used after incoming AXFR transfers, see :ref:`modes-of-operation-axfrend`
```
